src/components/Image/Image.visual.tsx

```diff
@@ -164,7 +168,7 @@ visualize('Image', () => {
       story('with wiring', () => {
         snap('override background color', (done) => (
           <ImageWithWrapper
-            src={invalidSrc}
+            src={src}
```

Comment:
oh I thought the invalidSrc was to test the error state, no?

Reply:
Yep it is and it's still being used for the error state. But I also used it to test the background color wiring.